### PR TITLE
Make branch name configurable

### PR DESCRIPTION
**Pull request by dharanikumar**
**Affects:** \<gitlab-collector-version-number>.

* [Individual Agreement](https://docs.google.com/forms/d/19LpBBjykHPox18vrZvBbZUcK6gQTj7qv1O5hCduAZFU/viewform)
* [Corporate Agreement](https://docs.google.com/forms/d/e/1FAIpQLSeAbobIPLCVZD_ccgtMWBDAcN68oqbAJBQyDTSAQ1AkYuCp_g/viewform?usp=send_form)

- [ ] I Have signed the CLA

---

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -38,6 +38,9 @@ gitlab.buildStages = build, prod, stage, deploy, build, test, analysis, docker,
 #Project ids
 gitlab.projectIds=projectId1,projectId2
 
+#Repo names
+gitlab.branchNames=repoName1,repoName2
```
Comment (kumarsi) [inline src/main/resources/application.properties]: Should be branchName1, branchName2

**Review by kumarsi — CHANGES_REQUESTED** [review]
Get rid of the considerOnlyMasterBuilds setting

File: src/main/resources/application.properties (hunk 1 above)
Comment (dharanikumar) [reply src/main/resources/application.properties]: it was a miss. renamed it. Thanks

**Comment by dharanikumar** [comment]
> Get rid of the considerOnlyMasterBuilds setting

good catch. removed it

**Pull request merged**